dist/index.html

```diff
@@ -2,10 +2,11 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <!-- <meta name="viewport" content="width=device-width, initial-scale=1.0"> -->
```

Comment:
- [ ] It's against the best practices to leave unused codes as comments on your codebase. I'll suggest you remove this. :+1: :+1: